cmd/agent/dist/conf.d/systemd.d/conf.yaml.example

```diff
@@ -0,0 +1,33 @@
+init_config:
+
+instances:
+    -
+    ## @param unit_names - list of strings - optional
+    ## List of systemd units to monitor (full name e.g. ssh.service).
+    ## If unit_names and unit_regexes are both NOT specified, the check will monitor all units.
+    #
+    # unit_names:
+    #   - <UNIT_NAME>
+
+    ## @param unit_regexes - list of strings - optional
+    ## Regex pattern[s] matching the names of units to monitor.
+    ## If unit_names and unit_regexes are both NOT specified, the check will monitor all units.
+    ## 
+    ## If the number of units managed by systemd is unbounded, it might infinitely increase
+    ## the number of metrics.
+    ## Learn more about custom metrics: https://docs.datadoghq.com/developers/metrics/custom_metrics/#how-is-a-custom-metric-defined
+    #
+    # unit_regexes:
```

Comment:
It seems that the examples below follow a regex logic (hence the name :p ) I think we should document those exemple above in the desc and leave only one placeholder as default value.

Reply:
Left one example only, I guess regex is a well known technology, that should be enough :)